Configure

```diff
@@ -1481,7 +1481,7 @@ $cpuid_obj.=" uplink.o uplink-x86.o" if ($cflags =~ /\-DOPENSSL_USE_APPLINK/);
 if ($target =~ /\-icc$/)	# Intel C compiler
 	{
 	my $iccver=0;
-	if (open(FD,"$cc -V 2>&1 |"))
+	if (open(FD,"$cross_compile_prefix$cc -V 2>&1 |"))
```

Comment:
On *side* note this can't actually happen, there is no such thing as icc cross-compiler. But for consistent view...